app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -3,7 +3,8 @@ import { Controller } from "stimulus";
 export default class extends Controller {
   static targets = ["title", "titleField",
                     "file", "fileField",
-                    "keywordsField", "contributorsField"];
+                    "keywordsField", "contributorsField",
+                    "embargo_dateField"];
```

Comment:
given that https://github.com/sul-dlss/happy-heron/pull/488/files#diff-b4ce442316cd2ea232e97f158d51b8e9b60077a59e600f7a1b76191fcf33b841R14 is `embargo-dateField`, shouldn't this match?

Reply:
And, not particularly important, but `embargoDateField` would be more aesthetically pleasing.

Reply:
I can try that, but this was what we discussed flipping between the javascript magic and the rails magic...

Reply:
There is no magic here.  These names just need to match what you wrote in the HTML.

Reply:
Yeah, that works, fixed.

Reply:
My first pass at renaming broke this, I'll take another swipe, but wouldn't hold this up over it myself.